mono_repo/lib/src/github_config.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:json_annotation/json_annotation.dart';
+
+part 'github_config.g.dart';
+
+@JsonSerializable(createToJson: false, disallowUnrecognizedKeys: true)
+class GitHubConfig {
+  final Map<String, dynamic> on;
+
+  // This is currently just to make JsonSerializable happy – need to fix!
```

Comment:
Add a TODO with a bug link please so this can be cleaned up more easily, or if you don't think it will be cleaned up then just mention it is a workaround.